daq_macros.py

```diff
@@ -1867,17 +1871,23 @@ def snakeRasterBluesky(rasterReqID, grain=""):
     if govStatus.exception():
       logger.error(f"Problem during start-of-raster governor move, aborting! exception: {govStatus.exception()}")
       return
-    raster_flyer.configure_detector(file_prefix=rasterFilePrefix, data_directory_name=data_directory_name)
-    raster_flyer.detector.stage()
+    #raster_flyer.configure_detector(file_prefix=rasterFilePrefix, data_directory_name=data_directory_name)
+    #raster_flyer.detector.stage()
     procFlag = int(getBlConfig("rasterProcessFlag"))
     spotFindThreadList = []
+    yield from bps.mv(samplexyz.omega, (omega-1)) # attempting to over-compensate omega movement
     for row_index, row in enumerate(rows):  # since we have vectors in rastering, don't move between each row
-        xMotAbsoluteMove, xEnd, yMotAbsoluteMove, yEnd, zMotAbsoluteMove, zEnd = raster_positions(row, stepsize, omegaRad, rasterStartX, rasterStartY, rasterStartZ, row_index)
-        vector = {'x': (xMotAbsoluteMove, xEnd), 'y': (yMotAbsoluteMove, yEnd), 'z': (zMotAbsoluteMove, zEnd)}
+        logger.info(f'starting new row: {row_index}')
+        zMotAbsoluteMove, zEnd, yMotAbsoluteMove, yEnd, xMotAbsoluteMove, xEnd = raster_positions(row, stepsize, omegaRad+90, rasterStartZ*1000, rasterStartY*1000, rasterStartX*1000, row_index)
+        vector = {'x': (xMotAbsoluteMove/1000, xEnd/1000), 'y': (yMotAbsoluteMove/1000, yEnd/1000), 'z': (zMotAbsoluteMove/1000, zEnd/1000)}
+        yield from bps.mv(samplexyz.x, xMotAbsoluteMove/1000)
```

Comment:
If it is safe to move in parallel, these can all be put in a single `bps.mv(mtr, target, mtr, target, ...)` call which should save some time.

Reply:
Great idea, I have put that in with [59cb602](https://github.com/NSLS-II/lsdc/pull/314/commits/59cb602fd08c29ca22d01515ba775b17798eadb2)